llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -12989,29 +12990,41 @@ SDValue DAGCombiner::visitPARTIAL_REDUCE_MLA(SDNode *N) {
 //
 // partial_reduce_*mla(acc, mul(ext(x), splat(C)), splat(1))
 // -> partial_reduce_*mla(acc, x, C)
+//
+// partial_reduce_fmla(acc, fmul(fpext(a), fpext(b)), splat(1.0))
+// -> partial_reduce_fmla(acc, a, b)
+//
+// partial_reduce_fmla(acc, fmul(fpext(x), splat(C)), splat(1.0))
+// -> partial_reduce_fmla(acc, x, C)
```

Comment:
C in this context is a scalar so the result pattern doesn't make sense.  That said, I don't see this transform implemented below so the comment can be removed?